masonry/src/event_loop_runner.rs

```diff
@@ -13,16 +13,29 @@ use vello::{peniko::Color, AaSupport, RenderParams, Renderer, RendererOptions, S
 use wgpu::PresentMode;
 use winit::application::ApplicationHandler;
 use winit::error::EventLoopError;
-use winit::event::WindowEvent as WinitWindowEvent;
+use winit::event::{MouseButton as WinitMouseButton, WindowEvent as WinitWindowEvent};
 use winit::event_loop::{ActiveEventLoop, EventLoopProxy};
 use winit::window::{Window, WindowAttributes, WindowId};
 
 use crate::app_driver::{AppDriver, DriverCtx};
 use crate::dpi::LogicalPosition;
-use crate::event::{PointerState, WindowEvent};
+use crate::event::{PointerButton, PointerState, WindowEvent};
 use crate::render_root::{self, RenderRoot, WindowSizePolicy};
 use crate::{PointerEvent, TextEvent, Widget};
 
+impl From<WinitMouseButton> for PointerButton {
+    fn from(button: WinitMouseButton) -> Self {
+        match button {
+            WinitMouseButton::Left => PointerButton::Primary,
+            WinitMouseButton::Right => PointerButton::Secondary,
+            WinitMouseButton::Middle => PointerButton::Auxiliary,
+            WinitMouseButton::Back => PointerButton::X1,
+            WinitMouseButton::Forward => PointerButton::X2,
+            WinitMouseButton::Other(_) => todo!(),
```

Comment:
I think this is for mice with more than 5 buttons. For example, my mouse has a DPI up/down button (although these specifically don't actually use this).
It looks like some users have seen this actually be used, e.g. https://github.com/rust-windowing/winit/issues/2562

I'd prefer to use a `tracing::warn!("...")` here (maybe a `tracing::debug!`?) rather than a full `panic`; this isn't an error condition, just one we're not sure how to expose.

Reply:
I've added a `PointerButton::Other` for now (no value attached) and mapped to it here with a `tracing::warn!` emitted.